packages/frontend/src/components/claims/index.tsx

```diff
@@ -3,76 +3,104 @@
 import { useClaims } from "@/src/hooks/useClaims";
 import { Chains, ChainsSkeleton } from "./chains";
 import { SupportedChain } from "@metrom-xyz/contracts";
-import { type Claim } from "@metrom-xyz/sdk";
+import { type Claim, type Reimbursement } from "@metrom-xyz/sdk";
 import { useEffect, useMemo, useState } from "react";
 import { type Chain } from "viem";
 import { ChainOverview, SkeletonChainOverview } from "./chain-overview";
 import { ChainClaims, SkeletonChainClaims } from "./chain-claims";
 import { Empty } from "./empty";
 import { CHAIN_DATA, SUPPORTED_CHAINS, type ChainData } from "@/src/commons";
+import { useReimbursements } from "@/src/hooks/useReimbursements";
+import { ChainReimbursements } from "./chain-reimbursements";
 
 import styles from "./styles.module.css";
 
-export interface ChainWithClaimsData {
+enum RewardType {
+    claim = "claim",
```

Comment:
Can we have this in PascalCase?

Reply:
Addressed in cdfbc6eadb4ca76641cdd6c5020fdfc296b367ad.